### PR TITLE
Add README for BERT E2E example

### PR DESCRIPTION
**Pull request by chilo-ms**
Add README for users to better understand and use our e2e BERT QDQ example.

### CODE REVIEW

**Comment by jywu-msft** [comment]
would it be useful to add some details about the model evaluation results?
i.e. on T4, what perf/accuracy was obtained int8 vs original fp32 precision.
And also the test environment setup (software/hardware versions etc.)

**Comment by chilo-ms** [comment]
> it would be nice to add some details about the model evaluation results. goal of this would be help user to replicate same results we had. i.e. on T4, what perf/accuracy was obtained int8 vs original fp32 precision. And also the test environment setup (software/hardware versions etc.)

Will do

**Comment by chilo-ms** [comment]
@jywu-msft I just updated the README, please take a look when you have time.

File: quantization/nlp/bert/trt/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# BERT QDQ Quantization in ONNX for TensorRT  
+There are two main steps for the quantization:
+1. Calibration is done based on SQuAD dataset to get dynamic range of floating point tensors in the model
+2. Q/DQ nodes with dynamic range (scale and zero-point) are inserted to the model
+
+After Qunatization is done, you can evaluate the QDQ model by running evaluation script we provide or your own script.
```
Comment (jywu-msft) [inline quantization/nlp/bert/trt/README.md:6]: typo s/Qunatization/Quantization/

File: quantization/nlp/bert/trt/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## Performance
+The performance results were obtained by running [onnxruntime_perf_test](https://github.com/microsoft/onnxruntime/tree/master/onnxruntime/test/perftest) with TensorRT 8.2.2.3 on
+NVIDIA T4 with (1x T4 32G) GPUs. 
+```shell
+./onnxruntime_perf_test -e tensorrt -r 10000 qdq_model.onnx -o 0 -i 'trt_fp16_enable|true trt_int8_enable|true trt_engine_cache_enable|true'
+```
+
+
+The accuracy results were obtained by running e2e_tensorrt_bert_example.py as above mentioned. 
+#### BERT Base
```
Comment (jywu-msft) [inline quantization/nlp/bert/trt/README.md:54]: would be useful to include the FP32 results to compare against.

File: quantization/nlp/bert/trt/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+| Sequence Length | Batch Size | FP16 Latency (ms) |  INT8 Latency (ms) |  FP16 Accuracy (F1) |    INT8 Accuracy (F1) |
+|-----------------|------------|-----------------|--------|-------------|---------|
```
Comment (jywu-msft) [inline quantization/nlp/bert/trt/README.md:56]: do you also have fp32 numbers?

Comment (chilo-ms) [reply quantization/nlp/bert/trt/README.md:56]: I can run it and add them there.

Comment (chilo-ms) [reply quantization/nlp/bert/trt/README.md]: FP32 numbers added.

File: quantization/nlp/bert/trt/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+2. Q/DQ nodes with dynamic range (scale and zero-point) are inserted to the model
+
+After Quantization is done, you can evaluate the QDQ model by running evaluation script we provide or your own script.
+
+The **e2e_tensorrt_bert_example.py** is an end-to-end example for you to reference and run.
+
+## Requirements
+* Please build from latest ONNX Runtime source (see [here](https://onnxruntime.ai/docs/build/eps.html#tensorrt)) for now.
+We plan to include TensorRT QDQ support later in ONNX Runtime 1.11 for [ORT Python GPU Package](https://pypi.org/project/onnxruntime-gpu/)
+* TensorRT 8.2
```
Comment (stevenlix) [inline quantization/nlp/bert/trt/README.md:13]: make it more precise 8.2.2.1?

File: quantization/nlp/bert/trt/README.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+./onnxruntime_perf_test -e tensorrt -r 10000 qdq_model.onnx -o 0 -i 'trt_fp16_enable|true trt_int8_enable|true trt_engine_cache_enable|true'
+```
+
+
+The accuracy results were obtained by running e2e_tensorrt_bert_example.py as above mentioned. 
+#### BERT Base
+| Sequence Length | Batch Size | FP32 Latency (ms) | FP16 Latency (ms) |  INT8 Latency (ms) | FP32 Accuracy (F1) | FP16 Accuracy (F1) |    INT8 Accuracy (F1) |
+|-----------------|------------|----|-------------|--------|------|-------|---------|
+| 128 | 1 | 7.33134 ms| 2.14245 ms | 1.65864 ms |88.259| 88.111 | 82.988 |
+| 384 | 1 | 21.6183 ms| 4.55848 ms | 3.58168 ms |88.662| 88.595 | 82.739 |
```
Comment (stevenlix) [inline quantization/nlp/bert/trt/README.md:58]: It seems strange that accuracy of sequence length 384 is less than 128.

Comment (chilo-ms) [reply quantization/nlp/bert/trt/README.md]: I generated the two qdq models with sequence length 384 and sequence length 128 again. Still got the same accuracy numbers. However, when I increased the calibration number from 100 to 200, the accuracy number of sequence length 384 increases to almost 82.988. So I'm thinking with larger calibration number, sequence length 384 could have better accuracy than sequence length 128.

File: quantization/nlp/bert/trt/README.md (hunk 4 above)
Comment (chilo-ms) [reply quantization/nlp/bert/trt/README.md]: Changed to 8.2.2.1

**Review by jywu-msft — APPROVED** [review]

**Pull request merged**